Libplanet.Explorer/Queries/Query.cs

```diff
@@ -57,8 +57,11 @@ public Query(BlockChain<T> chain, IStore store)
 
                 if (isMinerValid && isTxValid)
                 {
-                    limit--;
-                    yield return block;
+                    if (offset-- <= 0)
+                    {
+                         limit--;
+                         yield return block;
+                    }
```

Comment:
I guess it may be more readable.
```suggestion
                    if (offset > 0)
                    {
                        offset--;
                    }
                    else
                    {
                         limit--;
                         yield return block;
                    }
```

Reply:
amend it